data-science-onramp/ai-platform/modules/trainer/sklearn_model/task.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START ai_platform_sklearn_task]
+# [START ai_platform_sklearn_task_imports]
+import argparse
+import os
+import re
+
+import joblib
+from sklearn.metrics import mean_absolute_error
+from google.cloud import storage
+
+from trainer import utils
+from trainer.sklearn_model import model
+# [END ai_platform_sklearn_task_imports]
+
+# [START ai_platform_sklearn_task_args]
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--input-path",
+        type=str,
+        required=True,
+        help="path to input data"
+    )
```

Comment:
same comment as tfkeras regarding arg parser - is this a requirement for AI Platform/scikit learn or can it be done w/o arg parsing?

Reply:
Ack above